src/libraries/Common/src/Interop/Windows/Crypt32/Interop.CryptUnprotectData.cs

```diff
@@ -9,15 +9,15 @@ internal static partial class Interop
 {
     internal static partial class Crypt32
     {
-        [DllImport(Libraries.Crypt32, CharSet = CharSet.Unicode, SetLastError = true)]
+        [GeneratedDllImport(Libraries.Crypt32, SetLastError = true)]
         [return: MarshalAs(UnmanagedType.Bool)]
-        internal static extern bool CryptUnprotectData(
-                  [In] ref DATA_BLOB pDataIn,
-                  [In] IntPtr ppszDataDescr,
-                  [In] ref DATA_BLOB pOptionalEntropy,
-                  [In] IntPtr pvReserved,
-                  [In] IntPtr pPromptStruct,
-                  [In] CryptProtectDataFlags dwFlags,
-                  [Out] out DATA_BLOB pDataOut);
+        internal static partial bool CryptUnprotectData(
+            ref DATA_BLOB pDataIn,
```

Comment:
Do we want to make this `in` instead of `ref` as the previous version had `[In]`?

Reply:
There are many of those throughout the code base. I'm not convinced we need to change them. If we see them and want to make things nicer I suppose it could be useful.